ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/management-gnmi-new-flags-novrf.md

```diff
@@ -0,0 +1,124 @@
+# management-gnmi
```

Comment:
you see deletions in the CI because you have not added these new "hosts" to the "hosts.ini" under molecule. So when the CI rerun it is not creating the config and documentation files. I am not sure you managed to get these files created without adding them to "hosts.ini"